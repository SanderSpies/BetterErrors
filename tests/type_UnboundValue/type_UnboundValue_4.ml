let poshanassy = 1
let poshanessy = 2

let () = print_int poshanossy
