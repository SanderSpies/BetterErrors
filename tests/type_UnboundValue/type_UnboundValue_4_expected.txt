[36mtests/type_UnboundValue/type_UnboundValue_4.ml:4 19-29
[0m1 | let poshanassy = 1
2 | let poshanessy = 2
3 | 
4 | let () = print_int [31;4mposhanossy[0m

`poshanossy` can't be found. Did you mean `poshanessy or poshanassy`?
